package.json

```diff
@@ -8,19 +8,19 @@
     "start": "next start",
     "lint": "next lint",
     "test": "jest",
-    "test:watch": "jest --watch"
+    "test:watch": "jest --watch",
+    "format": "./node_modules/.bin/eslint -c 'eslint.config.mjs' .",
+    "format:fix": "./node_modules/.bin/eslint -c 'eslint.config.mjs' --fix ."
   },
   "engines": {
-    "node": ">=18.0.0"
+    "node":"^18.18.0 || ^20.9.0 || >=21.1.0"
```

Comment:
How come we switch the node 18 dep to use the `^` as opposed to the `>=` qualifier?

Reply:
Because the upgrade is only compatible with version 18.18, 20.9, or anything greater than version 21

Reply:
We discussed and we're using ^20.9.0

Reply:
Based on our new understanding the `^` means all minor and patch versions stopping at the next major version. Given that, and the fact that we don't really probably need to be on node 18 any more and more over we should be careful relying on odd versions of node since they seem to have a very short lifecycle. [node versions](https://nodejs.org/en/about/previous-releases)